Assets/Scripts/Projects/ProjectData.cs

```diff
@@ -3,85 +3,92 @@
 using System.Runtime.InteropServices;
 using ICSharpCode.SharpZipLib.Core;
 using ICSharpCode.SharpZipLib.Zip;
-using Netherlands3D.Twin.Functionalities;
 using Netherlands3D.Twin.Layers;
-using Netherlands3D.Twin.UI.LayerInspector;
 #if UNITY_EDITOR
 using UnityEditor;
 #endif
 using UnityEngine;
 using UnityEngine.Events;
 using Newtonsoft.Json;
-using Application = UnityEngine.Application;
+using UnityEngine.Assertions;
 
 namespace Netherlands3D.Twin.Projects
 {
     [CreateAssetMenu(menuName = "Netherlands3D/Twin/Project", fileName = "Project", order = 0)]
+    [Serializable]
     public class ProjectData : ScriptableObject
     {
-        private static ProjectData current;
-        public static ProjectData Current => current;
-        
-        [DllImport("__Internal")] private static extern void DownloadFromIndexedDB(string filename, string callbackObjectName, string callbackMethodName);
-        [DllImport("__Internal")] private static extern void SyncFilesToIndexedDB(string callbackObjectName, string callbackMethodName);
-        
+        [JsonIgnore] private static ProjectData current;
+        [JsonIgnore] public static ProjectData Current => current;
+        [JsonIgnore, NonSerialized] private bool isLoading = false; //is the project data currently loading? if true don't add the Layers to the root's childList, because this list is stored in the json, if false, a layer was created in app, and it should be initialized 
+
+        [DllImport("__Internal")]
+        private static extern void DownloadFromIndexedDB(string filename, string callbackObjectName, string callbackMethodName);
+
+        [DllImport("__Internal")]
+        private static extern void SyncFilesToIndexedDB(string callbackObjectName, string callbackMethodName);
+
         public const string DefaultFileName = "NL3D_Project_";
         public const string ProjectFileExtension = ".nl3d";
         public const string ProjectJsonFileNameInZip = "project.json";
-        
-        [Header("Serialized data")]
-        public int Version = 1;
+
+        [Header("Serialized data")] public int Version = 1;
         public string SavedTimestamp = "";
         public string UUID = "";
         public double[] CameraPosition = new double[3]; //X, Y, Z,- Assume RD for now
```

Comment:
As discussed on Slack: don't assume RD but always use a CRS. Assuming RD means we MUST make a BC breaking change when we want to support coordinates outside of the Netherlands. And we have a coordinate system that fully supports this, so there is no need for such a shortcut

Reply:
This was already here, and not part of this PR, I can pick this up when saving the ObjectLibrary objects, as these need to save their position as a coordinate, and I can change the CameraPosition at the same time if needed

Reply:
If you would, yes please! That would be wonderful :)